libkbfs/prefetcher.go

```diff
@@ -9,15 +9,24 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/keybase/client/go/logger"
+	"github.com/pkg/errors"
+
```

Comment:
`goimports` does this by default, which is my run-on-save script to avoid having to worry about imports. I've edited it, although I'm not sure we should be picky about things like that.